src/wp-includes/comment-template.php

```diff
@@ -1842,8 +1851,8 @@ function get_comment_reply_link( $args = array(), $comment = null, $post = null
 				)
 			) . '#' . $args['respond_id'],
 			$data_attribute_string,
-			esc_attr( sprintf( $args['reply_to_text'], get_comment_author( $comment ) ) ),
-			$args['reply_text']
+			$aria_label ? ' aria-label="' . esc_attr( $aria_label ) . '"' : '',
+			esc_html( $reply_text )
```

Comment:
```suggestion
			$reply_text
```

`esc_html()` would break the `$reply_text` in themes such as [Twenty Eleven](https://github.com/WordPress/wordpress-develop/blob/b17b5d18548b7d9a1083425f2398322e0b82d613/src/wp-content/themes/twentyeleven/functions.php#L814) and [Twenty Seventeen](https://github.com/WordPress/wordpress-develop/blob/b17b5d18548b7d9a1083425f2398322e0b82d613/src/wp-content/themes/twentyseventeen/comments.php#L62)